frontend/src/views/MauDetailsContent.js

```diff
@@ -19,7 +19,7 @@ class MauDetailsContent extends Component {
         <li key={index} className={styles['content-row']}>
           <span className={styles['period']}>{period.period}</span>
           <span className={styles['mau-count']}>{period.value}</span>
-          <span className={cx({ 'difference': true, 'positive': (difference > 0), 'negative': (difference < 0)})}>{difference}</span>
+          <span className={cx({ 'difference': true, 'positive': (difference > 0), 'negative': (difference < 0)})}>{(difference > 0) ? "+" : ""}{difference}</span>
```

Comment:
Can we make this sign an emoji like :arrow_up_small: and :arrow_down_small:

Reply:
@OmarIthawi we can, but I'd rather do it in FontAwesome than (ugly) emoji :) Let's keep this off for now - I don't think it's a priority.